scripts/base/frameworks/analyzer/dpd.zeek

```diff
@@ -73,7 +72,7 @@ event analyzer_violation(c: connection, atype: AllAnalyzers::Tag, aid: count,
 		return;
 
 	delete c$service[analyzer];
-	add c$service[fmt("-%s", analyzer)];
+	add c$service_violation[atype];
```

Comment:
Yeah, can do. I was forth and back, because `c$service` should probably have been tags rather than strings, but we can't change that. Keeping it at least consistent makes more sense.